src/_data/decommissioned_projects.yml

```diff
@@ -0,0 +1,10 @@
+-
```

Comment:
Would be nice to see screenshots of what these new sections look like laid out together

Reply:
Honestly, I'm not happy with the overall layout but this whole section needs a new IA (including moving "how to become a project" onto its own page, probably along with decommissioned/event-based/expired projects).

![projects-screenshot](https://user-images.githubusercontent.com/32755410/97936740-5749f600-1d31-11eb-91f4-35e20b583366.png)